backend/canisters/user/CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Removed unused timer job type (`RetrySendingFailedMessage`) ([#3263](https://github.com/open-ic/open-chat/pull/3263))
 - Removed `affected_events` from event responses ([#3322](https://github.com/open-ic/open-chat/pull/3322))
 - Removed super_admin role from groups([#3319](https://github.com/open-ic/open-chat/pull/3319))
+- Remove owner_id from group summary ([#3340](https://github.com/open-ic/open-chat/pull/3340))
+
+### Changed
+
+- Cannot remove platform moderators from a group ([#3340](https://github.com/open-ic/open-chat/pull/3340))
```

Comment:
There is already a CHANGED section

Reply:
Also the description isn't very clear, I'm not sure what the change actually is.